src/App.css

```diff
@@ -26,3 +26,13 @@
     display: none;
   }
 }
+
+/* disable user text highlighting */
+.noselect {
+  -webkit-touch-callout: none; /* iOS Safari */
+  -webkit-user-select: none; /* Safari */
+  -khtml-user-select: none; /* Konqueror HTML */
+  -moz-user-select: none; /* Firefox */
+  -ms-user-select: none; /* Internet Explorer/Edge */
+  user-select: none; /* Non-prefixed version, currently supported by Chrome and Opera */
+}
```

Comment:
@4xdk Do you see this used anywhere?

Reply:
Ah nice spot. I've used it on the menu but it went missing during refactoring I'll add it back to nav-pursuances!